.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "bracketSpacing": false
+}
```

Comment:
We should be using single quotes to align with google styles.

https://prettier.io/docs/en/options.html#quotes

Also 'JSX brackets' to true

https://prettier.io/docs/en/options.html#jsx-brackets